src/main/java/baseEntities/BaseTest.java

```diff
@@ -0,0 +1,31 @@
+package baseEntities;
+
+import drivers.BrowserType;
+import drivers.DriverFactory;
+import drivers.DriverManager;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+
+    public static WebDriver getDriver() {
```

Comment:
why do we have getDriver() method inside BaseTest class? is it BaseTest functionality?

Reply:
fixed